dw-menu.js

```diff
@@ -143,53 +159,56 @@ export class DwMenu extends DwCompositeDialog {
     disableAutoClose: {
       type: Boolean,
     },
+
+    /**
+     * index of activated Item
+     * default: -1
+     */
+    _activatedIndex: { type: Number },
+
+
+    /**
+     * Activated item
+     * One of the Item from _items, by reference.
+     */
+    _activatedItem: { type: Object },
+
+      /**
+     * Filtered actions excluding hidden and disabled items
+     */
+    _filteredActions: { type: Array },
```

Comment:
@ujjval-kukadiya There is no need to define this property. Instead you can make it's getter.